MobileDevProject/Room.swift

```diff
@@ -0,0 +1,195 @@
+import Foundation
+import XLActionController
+
+open class RoomCell: ActionCell {
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        initialize()
+    }
+
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    open override func awakeFromNib() {
+        super.awakeFromNib()
+        initialize()
+    }
+
+    func initialize() {
+        backgroundColor = .clear
+        let backgroundView = UIView()
+        backgroundView.backgroundColor = UIColor.white.withAlphaComponent(0.3)
+        selectedBackgroundView = backgroundView
+        actionTitleLabel?.textColor = .white
+        actionTitleLabel?.textAlignment = .left
+
+    }
+}
+
+public struct RoomHeaderData {
+
+    var title: String
+    var subtitle: String
+//    var image: UIImage
```

Comment:
At first, I am thinking about if we should have an image in the ActionController on the left. It would look nice if you can have a picture of the room. Don't know what about others think.